seclang/parser.go

```diff
@@ -114,6 +117,9 @@ func (p *Parser) FromString(data string) error {
 			linebuffer.Reset()
 		}
 	}
+	if inBackticks {
+		return errors.New("backticks left open")
```

Comment:
Do you mind backfilling a test for this aswel?

Reply:
Sure, sorry. Can just something [like this](https://github.com/corazawaf/coraza/blob/df9ca93d25af279331894ace4d68a7bd2edbf478/seclang/parser_test.go#L55-L62) work? It is all about opening an (also legit) backtick and not closing it.